src/libstd/ffi/os_str.rs

```diff
@@ -173,6 +173,14 @@ impl ops::Deref for OsString {
     }
 }
 
+#[stable(feature = "rust1", since = "1.9.0")]
```

Comment:
needs a new feature name. For example `osstring_default` (use same for both). Version 1.9.0 is ok I think since we are still in version 1.9.0 in the nightly.